main/typings.d.ts

```diff
@@ -1,19 +1,22 @@
 export type ActivitySource = 'Station' | 'Saturn';
-export type ActivityEventType = 'info' | 'error';
+export type ActivityType = 'info' | 'error';
 
-export interface ActivityEvent {
-  type: ActivityEventType;
+export interface Activity {
+  id: string;
+  timestamp: number;
+  type: ActivityType;
   source: ActivitySource;
   message: string;
 }
 
-export interface ActivityEntry extends ActivityEvent {
-  id: string;
-  timestamp: number;
+export interface RecordActivityOptions {
+  type: ActivityType;
+  source: ActivitySource;
+  message: string;
```

Comment:
I removed the inheritance, because `Activity` is the main type, and `RecordActivityOptions` more a partial of it.

Reply:
It makes sense to remove inheritance 👍🏻 

An alternative approach that does not duplicate property definitions:

```ts
export type RecordActivityOptions = Pick<Activity, 'type' | 'source' | 'message'>;
```

Reply:
cool I like that!

Reply:
On the second thought, this is probably more future proof when it comes to adding new Activity props:

```ts
export type RecordActivityOptions = Omit<Activity, 'id' | 'timestamp'>;